elasticdl/python/elasticdl/layers/embedding.py

```diff
@@ -140,7 +141,7 @@ def _check_id_valid(self, ids):
                 % (first_may_exceed_id, self.input_dim)
             )
         if any(ids < 0):
-            raise ValueError(
+            logger.warning(
                 "The embedding id cannot be less than zero. "
```

Comment:
Here it says cannot be zero. Need to modify the message to tell users what could go wrong if id < 0